app/src/main/java/com/tugraz/quizlet/frontend/ScoreFragment.kt

```diff
@@ -42,6 +44,9 @@ class ScoreFragment : Fragment(){
 
         val scoreTextView = view.findViewById<TextView>(R.id.text_view_score)
         scoreTextView.text = score?.toString()
+
+        val highScoreTextView = view.findViewById<TextView>(R.id.text_view_high_score)
+        highScoreTextView.text = highscore?.toString()
```

Comment:
wouldn't that lead to an NPE if highscore is null?

Reply:
highscore should always be either something or 0